greenswitch/esl.py

```diff
@@ -1,12 +1,16 @@
 # Gevent imports
 import gevent
+import sys
 from gevent.queue import Queue
 import gevent.socket as socket
 from gevent.event import Event
 import logging
 import pprint
 
-import urllib
+if sys.version_info > (3,):
+    from urllib.parse import unquote
+else:
+    from urllib import unquote
```

Comment:
Hi @italorossi Yes, I know :). I did that way because my intention as not add other  dependency. 

What do you prefer?
 a) Use six
 b) create a module called compat.py for this case.